.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,44 @@
+# Thank you for contributing to iTwinUI by creating an issue!
+# Make sure you first do a search for existing issues so that this is is not a dulicate
+# Also visit our FAQ https://github.com/iTwin/iTwinUI/wiki/FAQ
+
+name: 🐞 Bug
+description: Create a bug report to help us improve
+title: ''
+labels: bug
+assignees: ''
```

[Blank strings are not allowed](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms#key-must-be-a-string). Maybe you meant to use empty array.

```suggestion
assignees: []
```